### PR TITLE
Check in/47007/travel pay page content

### PR DESCRIPTION
**Pull request by brianseek**
## Description
This PR adds the 4 travel pay question pages along with their tests. Also this PR handles setting the answers to the questions to the reducer in the form.data object.

## Original issue(s)
department-of-veterans-affairs/va.gov-team#47007

## Testing done
Unit and E2E tests

### CODE REVIEW

File: src/applications/check-in/day-of/pages/TravelMileage.jsx [hunk 1]
```diff
@@ -2,34 +2,28 @@ import React from 'react';
...
-        type="button"
-      >
-        {t('no')}
-      </button>
-    </Wrapper>
+    <TravelPage
+      header={t('are-you-claiming-only-mileage-for-your-trip')}
+      bodyText={
+        <p>
+          {t(
```
Comment (brianseek) [inline src/applications/check-in/day-of/pages/TravelMileage.jsx:16]: prettier fights against this one and blocks the commit, so it wins out.

File: src/applications/check-in/day-of/pages/TravelQuestion.jsx [hunk 2]
```diff
@@ -1,35 +1,34 @@
...
-        type="button"
-      >
-        {t('no')}
-      </button>
-    </Wrapper>
+    <TravelPage
+      header={t('would-you-like-to-file-a-travel-reimbursement-claim-now')}
+      bodyText={
+        <p>
+          {t(
```
Comment (brianseek) [inline src/applications/check-in/day-of/pages/TravelQuestion.jsx:23]: prettier fights against this one and blocks the commit, so it wins out.

File: src/applications/check-in/components/pages/TravelPage/index.jsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+      </Wrapper>
+    </>
+  );
+};
+TravelPage.propTypes = {
+  header: PropTypes.string.isRequired,
+  router: PropTypes.object.isRequired,
+  bodyText: PropTypes.node,
+  helpText: PropTypes.node,
+  pageType: PropTypes.string,
```
Comment (brianseek) [inline src/applications/check-in/components/pages/TravelPage/index.jsx:79]:
```suggestion
  pageType: PropTypes.string.isRequired,
```

**Review by acrollet — APPROVED** [review]
we did collab review and this is looking great

**Pull request merged**